### DISCUSSION TITLE
RTCGQ11LM (motion sensor) sending temperature.

### DISCUSSION THREAD

**Discussion by 1igeza** [Koenkk/zigbee2mqtt#5914] [Devices]
My all RTCGQ11LM (motion sensors) are sending temperature and this temperature is incorrect. 
`Jan 26 08:13:29 zigbee2mqt npm[10765]: Zigbee2MQTT:debug 2021-01-26 08:13:29: Received Zigbee message from 'Salon Czujnik Ruchu', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{"measuredValue":19}' from endpoint 1 with groupID 0
Jan 26 08:13:29 zigbee2mqt npm[10765]: Zigbee2MQTT:debug 2021-01-26 08:13:29: Received Zigbee message from 'Salon Czujnik Ruchu', type 'attributeReport', cluster 'msOccupancySensing', data '{"occupancy":1}' from endpoint 1 with groupID 0
Jan 26 08:13:29 zigbee2mqt npm[10765]: Zigbee2MQTT:info  2021-01-26 08:13:29: MQTT publish: topic 'zigbee2mqtt/Salon Czujnik Ruchu', payload '{"battery":100,"illuminance":19,"illuminance_lux":19,"last_seen":"2021-01-26T08:13:29+01:00","linkquality":66,"no_occupancy_since":0,"occupancy":true,"temperature":23,"voltage":3015}'
`
Does this motion sensors have temperature sensors?

**Comment by Koenkk** [comment]
They do! https://github.com/Koenkk/zigbee-herdsman-converters/pull/2054
